### PR TITLE
Feature/fix soft delete error

### LINKED ISSUE #21: The model [App\User] does not support soft deleting.
```
The model [App\User] does not support soft deleting.
```
https://github.com/spinen/laravel-garbage-man/blob/0bfed10e052f8659cab8dd9fdab4863ce70c081e/src/Commands/PurgeCommand.php#L169

Soft-deleting is enabled for this model. forceDelete() exists, but onlyTrashed() doesn't:
https://github.com/laravel/framework/blob/9b10dcb31f1cc81322f9ec193aea0a8ba9e6db88/src/Illuminate/Database/Eloquent/SoftDeletes.php#L29
(Laravel 5.5)

Perhaps we could delete this check?
```
!method_exists($model, 'onlyTrashed')
```
P.S.: I would do a pull request, but I'm not familiar with PHP packages.

### PR DESCRIPTION
**Pull request by ssfinney**
Fixes #21.

Removed the check for onlyTrashed() and removed the test that checked for that method.

I also cleaned up some test names.

### CODE REVIEW

File: src/Commands/PurgeCommand.php [hunk 1]
```diff
@@ -169,7 +169,7 @@ protected function purgeExpiredRecordsForModel($model, $days)
             return false;
         }
 
-        if (!method_exists($model, 'onlyTrashed') || !method_exists($model, 'forceDelete')) {
```
Comment (jimmypuckett) [inline src/Commands/PurgeCommand.php:172]: @ssfinney Is this possible now that we are targeting later versions of Laravel?

Comment (ssfinney) [reply src/Commands/PurgeCommand.php:172]: @jimmypuckett Even if so, why try to check for both methods at all?

Comment (jimmypuckett) [reply src/Commands/PurgeCommand.php:172]: I am just making sure that this is the right thing to do with the discussion that we had in #21.

There is one of those methods that is for one version of Laravel & then another for another version.

I am just asking if we ware doing the right one now that we are dropping old versions

Comment (ssfinney) [reply src/Commands/PurgeCommand.php:172]: Oh ok, I understand now @jimmypuckett.

Yes we are. As of L >=5.5 the `onlyTrashed` method is gone, and the `forceDelete` is the right one to check.

**Pull request merged**